gandalf.js

```diff
@@ -5,72 +5,75 @@
  * for the extras/ plugins if they are loaded into a presentation.
```

Comment:
SCRIPT-Well done with the efficient error handling, it covers all the possible scenarios.